### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by saharza** [ManifestWebDesign/angular-gridster#373]
Does anyone knows if this directive is being used in production?

**Comment by smajl** [comment]
Yes, we use it in production at our company. And next time, don't use mysterious issue titles like "Question"... ;)

**Comment by saharza** [comment]
OK, Thanks.
